.travis.yml

```diff
@@ -5,3 +5,6 @@ rvm:
   - 2.0.0
   - 2.1.2
 script: bundle exec rspec
+before_install:
+  - gem update --system 2.4.8
+  - gem install bundler -v 1.7.14
```

Comment:
Similarly we don’t need to specify a particular version of Bundler here — `gem install bundler` is enough, and gives a more realistic picture of whether the code is currently working with what’s available.

Stretch goal: I’d also prefer this `.travis.yml` change in a separate commit, because it’s not really anything to do with the `respond_to?` stuff.